### PR TITLE
Fat-2659:C350602 Select Acquisition Method in Order Template

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

File: cypress/e2e/settings/orders/select-acquisition-method-in-order-template.cy.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    .then(userProperties => {
+      user = userProperties;
+    });
+    cy.login(user.username, user.password, { path:SettingsMenu.ordersOrderTemplatesPath , waiter: OrderTemplate.waitLoading });
+  });
+
+  after(() => {
+    Organizations.deleteOrganizationViaApi(organization.id);
+    AcquisitionMethods.deleteAcquisitionMethodViaAPI(acquisitionMethod.id)
+    Users.deleteViaApi(user.userId);
```
Comment (re-jas) [inline cypress/e2e/settings/orders/select-acquisition-method-in-order-template.cy.js:38]: cypress says that user.userId is undefined
try this 
16 line const user = {};
29 line .then((userProperties) => {
      user.username = userProperties.username;
      user.password = userProperties.password;
      user.userId = userProperties.userId;
    });

File: cypress/e2e/settings/orders/select-acquisition-method-in-order-template.cy.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  });
+
+    it('C350602 Select Acquisition Method in Order Template (thunderjet)', { tags: [TestType.criticalPath, devTeams.thunderjet] }, () => {
+      OrderTemplate.newTemplate();
+      OrderTemplate.fillTemplateInformationWithAcquisitionMethod(orderTemplateName, organization.name, acquisitionMethod.value);
+      OrderTemplate.saveTemplate();
+      OrderTemplate.checkTemplateCreated(orderTemplateName);
+    });
+});
```
Comment (re-jas) [inline cypress/e2e/settings/orders/select-acquisition-method-in-order-template.cy.js:47]: test fails in cypress

File: cypress/support/fragments/settings/orders/acquisitionMethods.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        body: acquisitionMethod,
+        isDefaultSearchParamsRequired: false,
+      }).then(response => response.body),
+  
+    deleteAcquisitionMethodViaAPI: (id) => cy.okapiRequest({
+        method: 'DELETE',
+        path: `orders/acquisition-methods/${id}`,
+        isDefaultSearchParamsRequired: false,
+      }),
+    };
```
Comment (re-jas) [inline cypress/support/fragments/settings/orders/acquisitionMethods.js:21]: please could you check expected indentations and format of all 3 files. there are errors that eslint throws

File: cypress/e2e/settings/orders/select-acquisition-method-in-order-template.cy.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/e2e/settings/orders/select-acquisition-method-in-order-template.cy.js:38]: It works without increasing the code

**Review by MaksimTerekhov — APPROVED** [review]

**Review by re-jas — APPROVED** [review]

**Pull request merged**